cmd/kubernetes_show.go

```diff
@@ -139,7 +139,7 @@ If you wish to use a custom format, the available fields are:
 					if version.Type == "deprecated" {
 						continue
 					}
-					if version.Version > kubernetesCluster.Version {
+					if kubernetesCluster.Version != version.Version && version.Type == "stable" {
 						versionsList = append(versionsList, version.Version)
```

Comment:
I don't think we should restrict them to "stable" versions. There could be development versions available

Reply:
What do you suggest the fix to be then? 🤔

Reply:
Something like `if kubernetesCluster.Version != version.Version && (version.Type == "stable" || version.Type == "development")` ?

Reply:
I don't think this bit matters: `(version.Type == "stable" || version.Type == "development")` . There are only three types possible: "development", "stable" and "deprecated". We're doing a continue on "deprecated" . So I don't think this is the fix